.vscode/settings.json

```diff
@@ -8,7 +8,7 @@
     "**/build": true
   },
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```

Comment:
nice. vscode was also changing this property for me 👍

Reply:
let's keep it then and not fight 😅